shared/src/commonMain/kotlin/com/kcc/kmmhackathon/shared/network/FirebaseAuthenticationApi.kt

```diff
@@ -2,14 +2,16 @@ import com.kcc.kmmhackathon.shared.entity.FirebaseAuthenticationResponse
 import io.ktor.client.HttpClient
 import io.ktor.client.features.*
 import io.ktor.client.request.*
+import io.ktor.client.statement.HttpStatement
+import io.ktor.client.statement.readText
```

Comment:
I might be missing something but are these needed?

Reply:
These were needed for the error handler..if I removed them then the project wouldn't build.

Reply:
Ok cool 👍